classroom.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './student'
+
+class Classroom
+  attr_accessor :label
+

```

Comment:
- [ ] **Required:** Kindly add a getter for the students' instance variable. Consider the code snippet below:
`attr_reader :students`